darasimiolaifa/epicmail
# fix(UI): fixes bug in the User Interface of the app

This PR fixes bugs found in the UI by carrying out the following activities:

- set height of body element using viewport height to keep footer element down
- create page to show group details when clicked
- create a clearer flow of interfaces when creating group
- fix wrong navigation text in tabs in restricted pages
- add year to date column in inbox
- do general UI cleaning

To test, take the following steps:
- git clone https://github.com/darasimiolaifa/epicmail.git
- cd epicmail
- npm run dev
- Wait for the app to start, showing you the port it is listening on (e.g 3000)
- Go to your browser and visit localhost:3000
- Go through each of the pages and observe the changes
